src/ngRoute/route.js

```diff
@@ -206,8 +206,7 @@ function $RouteProvider() {
         var star = option === '*' ? option : null;
         keys.push({ name: key, optional: !!optional });
         slash = slash || '';
-        return ''
-          + (optional ? '' : slash)
+        return (optional ? '' : slash)
```

Comment:
`slash` is always a string, therefore  `(optional ? '' : slash)` is always a string and no string conversion is required.